pkg/machine/e2e/basic_test.go

```diff
@@ -19,32 +19,40 @@ var _ = Describe("run basic podman commands", func() {
 		teardown(originalHomeDir, testDir, mb)
 	})
 
-	It("Basic ops", func() {
-		name := randomString()
-		i := new(initMachine)
-		session, err := mb.setName(name).setCmd(i.withImagePath(mb.imagePath).withNow()).run()
-		Expect(err).To(BeNil())
-		Expect(session).To(Exit(0))
-
-		bm := basicMachine{}
-		imgs, err := mb.setCmd(bm.withPodmanCommand([]string{"images", "-q"})).run()
-		Expect(err).To(BeNil())
-		Expect(imgs).To(Exit(0))
-		Expect(len(imgs.outputToStringSlice())).To(Equal(0))
-
-		newImgs, err := mb.setCmd(bm.withPodmanCommand([]string{"pull", "quay.io/libpod/alpine_nginx"})).run()
-		Expect(err).To(BeNil())
-		Expect(newImgs).To(Exit(0))
-		Expect(len(newImgs.outputToStringSlice())).To(Equal(1))
-
-		runAlp, err := mb.setCmd(bm.withPodmanCommand([]string{"run", "quay.io/libpod/alpine_nginx", "cat", "/etc/os-release"})).run()
-		Expect(err).To(BeNil())
-		Expect(runAlp).To(Exit(0))
-		Expect(runAlp.outputToString()).To(ContainSubstring("Alpine Linux"))
-
-		rmCon, err := mb.setCmd(bm.withPodmanCommand([]string{"rm", "-a"})).run()
-		Expect(err).To(BeNil())
-		Expect(rmCon).To(Exit(0))
-	})
+	/*
+	 * FIXME: #15347 - Basic ops fails on PR runs and on x86_64
+	 * https://github.com/containers/podman/issues/15347
+	 */
+
+	/*
+		It("Basic ops", func() {
```

Comment:
Please don't just comment it all out; use `Skip("FIXME: #15347 - ssh known hosts broken")` instead

Reply:
tried that already but it gave me a bunch of validation errors. I can look further but I thought it was much simpler that way. I'll look further.